components/outcomes/ScalarDisputeBox.tsx

```diff
@@ -83,17 +87,31 @@ const ScalarDisputeBox = ({
       );
     },
     {
+      onBroadcast: () => {},
       onSuccess: () => {
-        notificationStore.pushNotification("Outcome Disputed", {
-          type: "Success",
-        });
-        onSuccess?.();
+        if (onSuccess) {
+          onSuccess?.({
+            type: market.scalarType as ScalarRangeType,
+            scalar: new Decimal(scalarReportValue).mul(ZTG).toFixed(0),
```

Comment:
Is it correct to multiply here? I thought it would be the other way around

Reply:
Yeah I think you are right since the `scalarReportValue` is the raw value from the input I think